### ISSUE TITLE
Doesn't work on https://towardsdatascience.com/

### ISSUE THREAD

**Issue by SpaceMonkeyForever** [mesuutt/read-medium-extension#5]
I changed manifest.json to add that domain but the site goes blank when I click load.

Example https://towardsdatascience.com/bert-nlp-how-to-build-a-question-answering-bot-98b1d1594d7b

**Comment by SpaceMonkeyForever** [comment]
when I debug, I can see that there's no response received when making the GET request

**Comment by mesuutt** [comment]
Hi @SpaceMonkeyForever, 

Are you using ad blocker? Maybe ad blocker blocks  ajax request that made when you pressed to the button.

**Comment by SpaceMonkeyForever** [comment]
@mesuutt I disabled my adblocker and it didn't make a difference.

**Comment by Diego74965** [comment]
I'm having the same problem with https://medium.datadriveninvestor.com/*

**Comment by mesuutt** [comment]
Yes, not working on some sites, but I couldn't find the solution.
